nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProvenanceReporter.java

```diff
@@ -445,12 +445,27 @@ public void modifyAttributes(final FlowFile flowFile) {
         modifyAttributes(flowFile, null);
     }
 
+    @Override
+    public void modifyAttributes(final FlowFile flowFile, final long processingMills) {
+        modifyAttributes(flowFile, null, -1L);
+        verifyFlowFileKnown(flowFile);
```

Comment:
If the line above is calling the method on line 461, I don't think this is necessary.

Reply:
I think you're correct, removed.